fabtests/common/shared.c

```diff
@@ -119,7 +119,7 @@ struct fi_rma_iov remote;
 
 struct ft_opts opts;
 
-struct test_size_param test_size[] = {
+struct test_size_param pre_test_size[] = {
```

Comment:
it is not clear what `pre` means. maybe just name the variable `all_test_size` because it is used by `-S all`?